devel/demangle/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        eklitzke demangle master
+version             20160204
+
+categories          devel
+license             GPL-3+
+maintainers         @catap korins.ky:kirill} openmaintainer
+
+description         Tool to demangle C++ symbols.
+long_description    This is a tool to help you demangle C++ symbols. \
+                    These are symbols that you might see while looking \
+                    at GDB backtraces, or perhaps as a result of looking \
+                    at the symbol table from nm, objdump, etc.
+
+checksums           rmd160  447c8800fe32be15bdf96cb1f1f9469f5cce459a \
+                    sha256  97910ba2796d0fbbfae94fa2526c497e27e986474ad40b4f3e1d87642258d864 \
+                    size    14843
+
+use_autoreconf      yes
+
+post-destroot {
+    xinstall -d ${destroot}${prefix}/share/doc/${name}/
+    xinstall -m 644 -W ${worksrcpath} LICENSE.txt README.md \
```

Comment:
Mode 644 is the default for non-executable files, so you can remove this too. Otherwise, looks good!

Reply:
> Mode 644 is the default for non-executable files, so you can remove this too. Otherwise, looks good!

I think, anyway. LOL

Change it, and double-check locally.